servicecontrol/creating-config-file.md

```diff
@@ -77,13 +77,17 @@ You can configure ServiceControl to forward any consumed messages into alternate
 
   The period that defines whether an endpoint is considered alive or not. Default: `00:00:40` (40 secs)
 
+* `ServiceControl/MaximumMessageThroughputPerSecond`

```

Comment:
@SzymonPobiega I didn't add the ExternalIntegrationsDispatchingBatchSize as you requested,  Can you add some detail to the MaximumMessageThroughputPerSecond.  

@SimonCropp's comment raises a good point the code doesn't impose a min/max for MaximumMessageThroughputPerSecond.   Should we and some sensible limits?